SU-SWS/stanford_profile
# D8CORE-5278 Added scheduler module and configured for all content types

# READY FOR REVIEW

# Summary
- D8CORE-5278 Added scheduler module and configured for all content types

# Need Review By (Date)
- 2/30

# Urgency
- medium

# Steps to Test
1. `composer require su-sws/stanford_profile:dev-D8CORE-5278 --no-update; composer update -n`
2. `blt drupal:update` or `blt di -n`
3. create a node and configure the scheduler to publish in the future. (1 minute would be enough)
4. wait for the time to pass
5. run cron `drush cron`
6. view your node and verify it is now published.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
